README.md

```diff
@@ -3,8 +3,18 @@
 ![main workflow](https://github.com/cancerDHC/ccdh-terminology-service/actions/workflows/main.yml/badge.svg)
 
 The CCDH terminology service is a RESTful API that supports the validation, lookup, binding, and mapping of the concepts 
-in the CCDH data harmonization process and the transformation and validation workflow.  
+in the CCDH data harmonization process and the transformation and validation workflow.
+# License:
+BSD License
```

Comment:
I usually pick MIT, but I'm not super picky. Any rationale for BSD over other licenses?